dbt/include/trino/macros/utils/timestamps.sql

```diff
@@ -0,0 +1,12 @@
+{% macro trino__current_timestamp() -%}
```

Comment:
Why plural name of the file `timestamps.sql`?

Reply:
I have tried to mimic as much the official dbt plugins so we can easily follow course in the next upgrades.